app/src/docs/AccountCard/AccountCardPageAdvanced.tsx

```diff
@@ -0,0 +1,216 @@
+/* @license Copyright 2023 @paritytech/polkadot-cloud authors & contributors
+SPDX-License-Identifier: GPL-3.0-only */
+
+import {
+  AccountCard,
+  ExtraComponentProps,
+  IconProps,
+} from "@packages/cloud-react/lib/recipes/AccountCard";
+import { Grid } from "@packages/cloud-react/lib/base/structure/Grid";
+import { Button } from "@packages/cloud-react/lib/buttons/Button";
+import { SimpleEditor } from "../lib/SimpleEditor";
+
+export const AccountCardPageAdvanced = () => {
+  const codeA = `
+import { AccountCard, IconProps, } from "@packages/cloud-react/lib/recipes/AccountCard";
+...
+const iconProps: IconProps = {
+  canCopy: true,
+  position: "right",
+  size: 3,
+  justify: "space-around",
+};
+...
+return (
+  <AccountCard icon={iconProps} address={"1f1yYj2bCFhJCTVdeWLDueUsrZynLAaj6jeMy18fjZ7Cr73"} ellipsis />
+);`;
+
+  const codeB = `
+import { AccountCard, IconProps, } from "@packages/cloud-react/lib/recipes/AccountCard";
+...
+const iconPropsLeft: IconProps = {
+  // canCopy is disabled thus icon is not clickable anymore
+  // position defaults to "left"
+  size: 2,
+  justify: "space-around",
+};
+...
+return (
+  <AccountCard icon={iconPropsLeft} address={"1f1yYj2bCFhJCTVdeWLDueUsrZynLAaj6jeMy18fjZ7Cr73"} ellipsis ellipsisAmount={2} />
+)`;
```

Comment:
Shall we just have quotes and not the curly braces for these `string` props?
```suggestion
  <AccountCard icon={iconPropsLeft} address="1f1yYj2bCFhJCTVdeWLDueUsrZynLAaj6jeMy18fjZ7Cr73" ellipsis ellipsisAmount={2} />
)`;
```